src/Symfony/Bundle/TwigBundle/Tests/DependencyInjection/Fixtures/php/extra.php

```diff
@@ -1,7 +1,8 @@
 <?php
 
 $container->loadFromExtension('twig', array(
-     'paths' => array(
-         'namespaced_path3' => 'namespace3',
-      ),
+    'paths' => array(
+     'namespaced_path3' => 'namespace3',
```

Comment:
The indentation of this line needs to be fixed.

Reply:
Fixed, thanks.